### PR TITLE
circulation: avoid duplicated loans requests

### LINKED ISSUE #24: As a patron, I cannot request a Loan on an Item on which I currently have a Loan
Patrons (librarians?) should not be able to create/request Loans on Items if the patron has currently a Loan for the same Item.

### PR DESCRIPTION
**Pull request by ntarocco**
* adds a check before creating a request to ensure that there is not
already a pending request or an active loan for the same patron on the
same item
* closes #24

### CODE REVIEW

**Comment by ntarocco** [comment]
Blocked by: https://github.com/inveniosoftware/invenio-circulation/pull/77

**Review by zzacharo — APPROVED** [review]

**Review by kpsherva — APPROVED** [review]

File: invenio_app_ils/config.py
Comment (ntarocco) [inline invenio_app_ils/config.py:488]: This is just to unblock the PR because I don't have time to fix it right now. I have created a ticket for this: https://github.com/inveniosoftware/invenio-app-ils/issues/47.

**Pull request merged**
